Library/Homebrew/formula_auditor.rb

```diff
@@ -37,6 +37,8 @@ def initialize(formula, options = {})
       @spdx_license_data = options[:spdx_license_data]
       @spdx_exception_data = options[:spdx_exception_data]
       @tap_audit = options[:tap_audit]
+      @previous = {}
```

Comment:
```suggestion
      @previous_committed = {}
```

or 

```suggestion
      @previous_version = {}
```

or something feels a little nicer here.

Reply:
`@previous_committed` makes sense to me. [We have a `:version` value in this hash, so `previous_version[:version]` would feel a bit weird/repetitious.]